docs/sources/configure-server/reference-configuration-parameters/index.md

```diff
@@ -123,6 +123,11 @@ pyroscopedb:
   # CLI flag: -pyroscopedb.retention-policy-min-disk-available-percentage
   [min_disk_available_percentage: <float> | default = 0.05]
 
+  # How long a block that has been uploaded to bucket storage should remain on
+  # disk
+  # CLI flag: -pyroscopedb.retention-policy-block-expiry
+  [block_expiry: <duration> | default = 10m]
```

Comment:
weird the default doesn't match

Reply:
I had to run `make reference-help` https://github.com/grafana/pyroscope/pull/2816/commits/daa1946007fe717df70abe28fb9e604e43a6d36f